Source/OCMock/NSMethodSignature+OCMAdditions.m

```diff
@@ -21,11 +21,56 @@
 
 @implementation NSMethodSignature(OCMAdditions)
 
-+ (NSMethodSignature *)signatureOfGetterForDynamicProperty:(NSString *)propertyName inClass:(Class)aClass
++ (NSMethodSignature *)signatureForDynamicPropertyMatchingSelector:(SEL)selector inClass:(Class)aClass
 {
+    BOOL isGetter = YES;
+    BOOL isDynamic = NO;
```

Comment:
Please don't change the code more than necessary. The declaration was perfectly fine further down.